index.coffee

```diff
@@ -146,12 +147,21 @@ class ResumableRequest extends stream.Readable
 			@bytesTotal ?= parseInt(response.headers['content-length'])
 			@_reportProgress()
 			@_reportProgress.flush()
-			@response = cloneResponse(response).on('data', @push.bind(this))
+			@response = cloneResponse(response)
+			@destinations.forEach ([ dest, opts ]) =>
+				@response.pipe(dest, opts)
+			delete @destinations
 			@emit('response', @response)
 
-		# wrapping the stream in a v3 Readable stream streamlines the emission
```

Comment:
What changed to remove the wrapping? The comment makes it sound important

Reply:
Wrapping the legacy stream helped with piping it’s contents which we don’t do anymore and as much as I tested it didn’t seem to introduce issues.

Reply:
Ok